arangod/RocksDBEngine/RocksDBVPackIndex.cpp

```diff
@@ -181,37 +268,95 @@ class RocksDBVPackUniqueIndexIterator final : public IndexIterator {
 
     _done = true;
 
+    if (_cache != nullptr) {
+      rocksdb::Slice key = lookupValueForCache();
+
+      for (size_t attempts = 0; attempts < 10; ++attempts) {
```

Comment:
What is the point in here trying 10 times and then further in in the `find` call using `getBucket` with `Cache::triesFast`? Should we maybe be spinning for longer in there? If it happens two often that we have contention here, then trying 10 times is not a solution.

Reply:
That is a very good point. I took the same approach here as is currently used in the RocksDBEdgeIndex.
We should probably remove the `for` loop around the calling code here and in RocksDBEdgeIndex, given that the inner `getBucket()` call will already try repeatedly.

Reply:
Addressed